README.md

```diff
@@ -32,6 +32,15 @@ python -m pip install "napari[all]"
 
 If you prefer conda over pip, you can replace the last line with: `conda install -c conda-forge napari`
 
+### containers:
+Docker
+```sh
+docker pull ghcr.io/napari/napari:main
+```
+Singularity
+```sh
+singularity exec -B /run oras://ghcr.io/napari/napari:latest napari -h
+```
```

Comment:
```suggestion
```

We're trying to keep some of the details out of the main readme now. These steps could instead be added in https://github.com/napari/napari/blob/main/docs/tutorials/fundamentals/installation.md#next-steps immediately above the final section. That can happen in a later PR